### ISSUE TITLE
feat: use conventional commit messages for start PR

### ISSUE THREAD

**Issue by manniL** [renovatebot/renovate#2658]
**What would you like Renovate to be able to do?**
Currently, the starting commit is called `Add renovate.json`. If the repository is using (and validating) conventional commit messages, the PR will fail because of that. I'd like to see conventional commit messages by default if they've been recognized, starting from the first PR.

**Describe the solution you'd like**
Change commit message of the first commit in the first PR to `chore: add renovate.json` or `build: add renovate.json`

**Additional context**
Example of a failing PR: https://travis-ci.org/Developmint/cipher-collection/jobs/441365342

**Comment by rarkins** [comment]
Thanks, this is a good idea. Looking up the conventional commits spec [here](https://www.conventionalcommits.org/en/v1.0.0-beta.2/#summary) it seems like `chore` would still be preferred.

**Comment by manniL** [comment]
@rarkins Right, `chore` would be the best solution that satisfies all users of conventional commits, not only those who use `angular` specs.
